pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -1618,6 +1618,43 @@
   yw_page_top_heading: "Young Women in Computer Science"
   yw_page_top_desc: "Become a CS Ambassador and be a part of closing the gender gap in computer science."
 
+  intl_pl_page:
```

Comment:
@mgc1194 using the new namespace pattern here and loooove it! So much easier to implement 🙌